src/pages/landing/Landing.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect, useState } from "react";
+import { useInterval } from "usehooks-ts";
+
+import { Container, Box, Typography, Button, TextField } from "@mui/material";
+import ConnectWallet from "components/ConnectWallet/ConnectWallet";
+
+import { useAppDispatch, useAppSelector } from "store/store";
+import { fetchPrice } from "./slices/price.slice";
+import { fetchTiers } from "./slices/tiers.slice";
+
+import type { Tier } from "./slices/tiers.slice";
+
+import "./Landing.css";
+
+const REFRESH_TIME = 30;
+
+const ConnectSection = (props: { onSubmit: () => void }) => (
+  <Box className="flex flex-row-reverse flex-1">
+    <Box className="flex flex-col justify-center m-0 min-w-[50vw] bg-white">
+      <Box className="text-center">
+        <Typography variant="h3" className="font-bold mb-4 tracking-[.2em]">
+          Testing Tool
+        </Typography>
+        <Typography variant="h6" className="font-normal mb-40 text-[#A6A7AD]">
+          Connect your in-browser wallet to login/sign-up
+        </Typography>
+        <Button variant="contained" size="large" className="py-3 px-4 normal-case font-medium" onClick={props.onSubmit}>
+          Connect your wallet
+        </Button>
```

Comment:
There are 2 buttons for connect wallet. Is it while we finish everything to have the two routes?

Reply:
Yes, I'm finishing the integration right now, it'll be fixed in the next commit